scanner/k8s-assets/scanner/scanner.go

```diff
@@ -112,6 +123,36 @@ func (s *Scanner) fetchImageId(pod v1.Pod, container v1.Container) string {
 	return ""
 }
 
+// extractImageInfo extracts image registry, image repository and the account name
+// from a container image
+func (s *Scanner) extractImageInfo(image string) (ImageInfo, error) {
+	// Split the string to remove the tag
+	parts := strings.Split(image, ":")
+	if len(parts) < 1 {
+		return ImageInfo{}, fmt.Errorf("invalid image")
+	}
+
+	// Split the remaining string by '/'
+	components := strings.Split(parts[0], "/")
```

Comment:
The term `component` has a special meaning in the Heureka context and it confused me on the first sight. Maybe we can rename it to something like `tokens`